core/src/impl/Kokkos_Error.hpp

```diff
@@ -79,6 +79,10 @@ namespace Impl {
 #define KOKKOS_IMPL_ABORT_NORETURN [[noreturn]]
 #endif
 
+#elif defined(KOKKOS_COMPILER_NVHPC)
+
+#define KOKKOS_IMPL_ABORT_NORETURN
+
```

Comment:
That should be taken care by the `else`, right.

Reply:
No that would go through the
```C++
#elif !defined(KOKKOS_ENABLE_OPENMPTARGET)
// Host aborts
#define KOKKOS_IMPL_ABORT_NORETURN [[noreturn]]